assets/stylesheets/default_messages.css

```diff
@@ -0,0 +1,71 @@
+/* Same style as part of scm.yml */
+table.filecontent {
+  border: 1px solid #e2e2e2;
+  border-collapse: collapse;
+  width: 98%;
+  background-color: #fafafa;
+}
+
+table.filecontent tbody {
+  font-family: Consolas, Menlo, "Liberation Mono", Courier, monospace;
+  font-size: 12px;
+}
+
+table.filecontent th {
+  border: 1px solid #e2e2e2;
+  background-color: #eee;
+}
+
+table.filecontent th.filename {
+  background-color: #e4e4d4;
+  text-align: left;
+  padding: 5px;
+}
+
+table.filecontent tr.spacing th {
+  text-align: center;
+}
+
+table.filecontent tr.spacing td {
+  height: 0.4em;
+  background: #EAF2F5;
+}
+
+table.filecontent th.line-num {
+  border: 1px solid #e2e2e2;
+  text-align: right;
+  width: 2%;
+  padding: 0 3px 0 0;
+  color: #999;
+  user-select: none;
+  -moz-user-select: none;
+  -o-user-select: none;
+  -ms-user-select: none;
+  -webkit-user-select: none;
+  font-weight: normal;
+}
+
+table.filecontent th.line-num a {
+  text-decoration: none;
+  color: inherit;
+}
+
+table.filecontent td.line-code {
+  padding: 0 0 0 4px;
+}
+
+table.filecontent td.line-code pre {
+  margin: 0px;
+  white-space: pre-wrap;
+  font-family: Consolas, Menlo, "Liberation Mono", Courier, monospace;
+  font-size: 12px;
+}
+
+table.filecontent tr:target th.line-num {
+  background-color: #E0E0E0;
+  color: #777;
+}
+
+table.filecontent tr:target td.line-code {
+  background-color: #DDEEFF;
+}
```

Comment:
Hide the menu bar style

e.g.
```suggestion
}

/* Hide unnecessary headers etc */
body.action-default_messages #sidebar,
body.action-default_messages #header,
body.action-default_messages #top-menu {
  display: none;
}

body.action-default_messages #main {
  padding: 0px;
}
```

Reply:
Do you need the following CSS.
```
body.action-default_messages #main {
 padding: 0px;
}
```

Reply:
When the screen is responsive, the following changes have been made to increase the height of the header.

```css
@media screen and (max-width: 899px)
#main {
    padding: 64px 0 0;
}
```
<img width="495" alt="ScreenShot-2019-06-13-11 08 34" src="https://user-images.githubusercontent.com/14245262/59398564-a35a6a80-8dcb-11e9-9e77-2cc37ea99f88.png">

The "body.action-default_messages #main" style is to eliminate extra space.

Reply:
> When the screen is responsive, the following changes have been made to increase the height of the header.

It confirmed. I think it's good.